android/src/main/cpp/headers/NativeProxy.h

```diff
@@ -60,7 +60,15 @@ class EventHandler : public HybridClass<EventHandler> {
       jni::alias_ref<react::WritableMap> event) {
     std::string eventAsString = "{NativeMap:null}";
     if (event != nullptr) {
-      eventAsString = event->toString();
+      try {
+        eventAsString = event->toString();
+      } catch (std::exception) {
```

Comment:
apparently `catch(folly::json::parse_error)` doesn't seem to catch the error @tomekzaw